### ISSUE TITLE
Batch size of 513x513

### ISSUE THREAD

**Issue by frodrigo** [tyndare/blur-persons#9]
The cropped and rescaled image size are now of 513x513, is it the expected result ?

Dezoom 3
```
search in box (0, 0, 1539, 1539) size 1539x1539
(513, 513)
search in box (0, 1246, 1539, 2785) size 1539x1539
(513, 513)
search in box (0, 2493, 1539, 4032) size 1539x1539
(513, 513)
search in box (742, 0, 2281, 1539) size 1539x1539
(513, 513)
search in box (742, 1246, 2281, 2785) size 1539x1539
(513, 513)
search in box (742, 2493, 2281, 4032) size 1539x1539
(513, 513)
search in box (1485, 0, 3024, 1539) size 1539x1539
(513, 513)
search in box (1485, 1246, 3024, 2785) size 1539x1539
(513, 513)
search in box (1485, 2493, 3024, 4032) size 1539x1539
(513, 513)
```

Dezzom 6
```
search in box (0, 0, 3024, 3078) size 3024x3078
(504, 513)
search in box (0, 954, 3024, 4032) size 3024x3078
(504, 513)
```
Is no more a square input. Not sure how it works. Does tensorflow scale, crop or refill the image to get the expect square size?

**Comment by tyndare** [comment]
I checked the original code and it was a size 513 that was used, not 512 as I initially thought. I tested with a size of 514 and the execution failed with the exception 
tensorflow.python.framework.errors_impl.InvalidArgumentError
     Expected size[2] in [0, 513], but got 514
so I assumed that 513 was indeed the correct network size and not 512.

With dezoom 6 you get an unsquared crop because your image height dezoomed 6 times, 3024/6 is 504 wich is lower than 513, this is not the case for the image width 4032/6 = 672 > 513.
I don't know what tensorflow is doing in this case but as far as I tested this is not an issue to run it in on unsquared images, even if probably not optimal.
